lib/Adapters/RegressionAdapter/RegressionAdapter.js

```diff
@@ -0,0 +1,445 @@
+    return url.format({
+        protocol : 'http',
+        hostname : hostname,
+        port : port,
+        pathname : path
+    });
+}
+
+TestHelper.prototype.initEntries = function() {
+
+    var that = this;
+    var root_mp4_path = config.get('rootFolderPath');
+    var p;
+    p = _.map(entriesInfo, function (entryConfig) {
+        // remove directory
+        var fullpath = root_mp4_path + '/' + entryConfig.entryId;
+        return fs_utils.cleanFolder(fullpath, 'obsolete').then(function () {
+            that.logger.debug('[%s] CLEANUP SUCCEEDED @@@@ removed content of path=%s', entryConfig.entryId, fullpath)
+
+            var flavors = entryConfig.flavorParamsIds.split(',');
+            that.entries[entryConfig.entryId] = new EntryInfo(entryConfig.entryId, entryConfig.firstChunklistIndex);
+            that.entries[entryConfig.entryId].data_warehouse_path = that.rootFolderPath + '/' + entryConfig.entryPath + '/' + entryConfig.entryId + '/' + flavors[0];
+
+            return that.getLastChunklistFileIndex(entryConfig.entryId, that.entries[entryConfig.entryId].data_warehouse_path)
+                .then(function (last_chunklist_index) {
+                    that.entries[entryConfig.entryId].last_chunklist_index = last_chunklist_index;
+                    that.validators[entryConfig.entryId] = new validator(entryConfig.entryId, flavors, that.entries[entryConfig.entryId].last_chunklist_index, entryConfig.validator);
+                })
+                .then(function() {
+                    return that.validators[entryConfig.entryId].init();
+                })
+                .then(function () {
+                    _.each(flavors, function (id) {
+                        that.entries[entryConfig.entryId].chunklist_index[id] = entryConfig.firstChunklistIndex;
+                        that.entries[entryConfig.entryId].read[id] = false;
+                    });
+                });
+        }).catch(function (error) {
+            that.logger.error('[%s] initialization failed. Error: %s, stack: %s', entryConfig.entryId, error.message, error.stack)
+        });
+    });
+    return Q.all(p);
+}
+
+TestHelper.prototype.initPlaylist = function() {
+
+    this.logger.debug("TestHelper initialization");
+
+    var that = this;
+
+    var p=_.map(entriesInfo, function(entryConfig) {
+       
+        var mediaServerPlaylistUrl = that.getFullLiveUrl(entryConfig.entryId, 'all');
+        var fullUrl = url.resolve(mediaServerPlaylistUrl, 'playlist.m3u8');
+
+        // read and parse the playlist and assemble playlist per flavor
+        return networkClient.read(fullUrl)
+            .then(function (content) {
+                that.initPlaylistPerFlavor(entryConfig.entryId, content);
+            })
+            .catch(function (error) {
+                that.logger.warn('Initialization of entryId failed with error %s. Failed to prepare playlist per flavor.', error);
```

Comment:
replaced in all places where error is printed to log with ErrorUtils.error2string(err).